src/robusta/core/reporting/base.py

```diff
@@ -147,6 +147,7 @@ def __init__(
         self.fingerprint = fingerprint
         self.starts_at = starts_at if starts_at else datetime.now()
         self.ends_at = ends_at
+        self.dirty = False
```

Comment:
As far as I can tell, you never set this to `True`

Reply:
You're right.
Adding it to event.add_finding